doorbell/package.json

```diff
@@ -2,22 +2,64 @@
   "name": "doorbell",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
+    "clean": "rimraf dist/",
+    "serve": "vue-cli-service serve",
+    "prebuild": "npm run clean",
+    "build": "vue-cli-service build",
+    "test": "npm run test:unit",
+    "test:unit": "vue-cli-service test:unit",
+    "lint": "vue-cli-service lint"
+  },
   "dependencies": {
-    "prettier": "^1.12.1",
-    "react": "^16.3.2",
-    "react-dom": "^16.3.2",
-    "react-scripts": "0.9.5",
-    "roslib": "^0.20.0"
+    "auto-ros": "^0.1.5",
+    "core-js": "^3.8.3",
+    "os-browserify": "^0.3.0",
+    "roslib": "^1.2.0",
+    "vue": "^3.2.13"
```

Comment:
Why vue? They have a pretty bad reputation for backwards compatibility compared to react